matlab-src/deprecated/README.txt

```diff
@@ -1,8 +0,0 @@
-More detailed documentation and comments throughout the code will be provided soon.
-
-To run the simulator, run the 'run_forwardDyn.m' file. User can change various parameters of the simulation by modifying 'set_parameters.m' file which can also be accessed from the GUI. GUI gives the options to set simulation duraion and step times which by default are set to 10 and 0.01 seconds relatively. Also the checkbox 'Plots' allows the plotting of the contact forces at the feet, joint torques and center of mass trajectory errors.
```

Comment:
There are two readmes for the controllers in master, but I already tought to rewrite them after the merge